pkg/agent/catalog/catalog.go

```diff
@@ -59,6 +60,7 @@ func BuiltIns() []catalog.Plugin {
 		na_azure_msi.BuiltIn(),
 		na_k8s_sat.BuiltIn(),
 		na_k8s_psat.BuiltIn(),
+		na_devid.BuiltIn(),
```

Comment:
`tpm_devid` sounds good to me! Do you believe that additionally, we should move the current `devid` directory into a parent `tpm` directory to follow package naming convention and to consider future `tpm` related attestors?   Similar to what we have in other plugins?  

e.g for kubernetes node attestors we have:

```bash
 plugin/{server/agent/common}/k8s/sat
plugin/{server/agent/common}/k8s/psat
```

In this case it would be:
```bash
 plugin/{server/agent/common}/tpm/devid
plugin/{server/agent/common}/tpm/...
```

Reply:
I have a small preference to not organize these into subdirectories. I kind of regret that choice with the k8s attestators, mostly because its nice to do `ls pkg/server/plugin/nodeattestor` and have an immediate idea of what plugins are available. I also doubt we'll have more than two, so the grouping seems a little overkill?

Reply:
It's also kinda nice having the folder name itself be the "name" of the plugin.